common/maplib/map.mjs

```diff
@@ -234,13 +235,33 @@ class GameMap {
     }
     return ret;
   }
-}
 
+  /**
+   * Get the Spawn Points from all maps.
+   * @return {mapCoord Array} : Random spawn points with mapCoord type
+  **/
+  getRandomSpawnPoint() {
```

Comment:
This function should return **a random spawn point** instead of an array of spawn points.